src/chrome/content/rules/Amica.com.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Amica.com">
+	<target host="www.amica.com" />
+	<target host="m.amica.com" />
```

Comment:
careers.amica.com
securemail.amica.com

Please also add a comment for the timeouts on futureleaders.amica.com and amica.com (without www).